codex-ui/src/vue/switch/Switch.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    :class="[$style.switch, disabled ? $style[`switch--disabled`] : '', switchValue ? $style[`switch--on`] : '']"
+    v-on="disabled === false ? { click: changeValue } : {}"
+  >
+    <div :class="[$style[`switch__ellipse`], $style[`switch__ellipse--${switchValue}`]]"></div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+const props = withDefaults(
+  defineProps<{
+    /**
+     * Value of the switch ('on' or 'off')
+     */
+    value?: boolean;
+
+    /**
+     * State of interaction with the component
+     */
+    disabled?: boolean;
+  }>(),
+  { value: true, disabled: false }
+);
+
+const switchValue = defineModel({ default: true });
```

Comment:
```suggestion
const switchValue = defineModel({ default: props.value });
```

Reply:
probably prop `value` is not needed anymore if we have `defineModel`. Need to check